Description: Reorder preference of video formats
Author: Rogério Brito <rbrito@ime.usp.br>
Forwarded: no
Last-Update: 2010-06-07

--- a/youtube-dl
+++ b/youtube-dl
@@ -685,7 +685,7 @@
 	_LOGIN_URL = 'http://www.youtube.com/signup?next=/&gl=US&hl=en'
 	_AGE_URL = 'http://www.youtube.com/verify_age?next_url=/&gl=US&hl=en'
 	_NETRC_MACHINE = 'youtube'
-	_available_formats = ['37', '22', '35', '18', '34', '5', '17', '13', None] # listed in order of priority for -b flag
+	_available_formats = ['37', '45', '22', '35', '18', '43', '34', '5', '17', '13', None] # listed in order of priority for -b flag
 	_video_extensions = {
 		'13': '3gp',
 		'17': 'mp4',
