Description: Give preference to open video formats
Author: Rogério Brito <rbrito@ime.usp.br>
Forwarded: no
Last-Update: 2010-10-13

--- a/youtube-dl
+++ b/youtube-dl
@@ -692,7 +692,7 @@
 	_AGE_URL = 'http://www.youtube.com/verify_age?next_url=/&gl=US&hl=en'
 	_NETRC_MACHINE = 'youtube'
 	# Listed in order of quality
-	_available_formats = ['38', '37', '22', '45', '35', '34', '43', '18', '6', '5', '17', '13']
+	_available_formats = ['38', '37', '45', '22', '43', '35', '34', '18', '6', '5', '17', '13']
 	_video_extensions = {
 		'13': '3gp',
 		'17': 'mp4',
